packages/grafana-ui/src/components/Table/FilterList.tsx

```diff
@@ -2,33 +2,131 @@ import { css, cx } from '@emotion/css';
 import React, { useCallback, useMemo, useState } from 'react';
 import { FixedSizeList as List } from 'react-window';
 
-import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { GrafanaTheme2, formattedValueToString, getValueFormat, SelectableValue } from '@grafana/data';
 
-import { Checkbox, FilterInput, Label, VerticalGroup } from '..';
+import { ButtonSelect, Checkbox, FilterInput, HorizontalGroup, Label, VerticalGroup } from '..';
 import { useStyles2, useTheme2 } from '../../themes';
 
 interface Props {
   values: SelectableValue[];
   options: SelectableValue[];
   onChange: (options: SelectableValue[]) => void;
   caseSensitive?: boolean;
+  showOperators?: boolean;
 }
 
 const ITEM_HEIGHT = 28;
 const MIN_HEIGHT = ITEM_HEIGHT * 5;
 
-export const FilterList = ({ options, values, caseSensitive, onChange }: Props) => {
+const operatorSelectableValue = (op: string): SelectableValue<string> => {
+  const result: SelectableValue<string> = { label: op, value: op };
+  switch (op) {
```

Comment:
This would make more sense as a string to string map in my opinion so that there's constant time lookup rather than it depending on how deep we go in the switch

Reply:
Thanks! code updated, I hope now looks better